test/functions/getFirstExistingParentPath.spec.ts

```diff
@@ -1,24 +1,44 @@
+import { normalize } from 'node:path'
+
 import test from 'ava'
 import { PathLike } from 'fs'
 
 import getFirstExistingParentPath from '@/src/functions/getFirstExistingParentPath'
 import mockDependencies from '@/test/__helpers__/mockDependencies'
 
+const getDependencies = (parentPath: string) => mockDependencies({
+	fsAccess: async (directoryPath: PathLike) => directoryPath === parentPath ? Promise.resolve() : Promise.reject(new Error('File does not exists')),
+})
 
 test('unix: get first existing parent path', async t => {
-	const parentPath = '/home/Alex'
-	const dependencies = mockDependencies({
-		fsAccess: async (directoryPath: PathLike) => directoryPath === parentPath ? Promise.resolve() : Promise.reject(new Error('File does not exists')),
-	})
+	const parentPath = normalize('/home/Alex')
```

Comment:
Why normalize the file path for a UNIX test since it's already a Unix-style path?

Reply:
True. Do you think we should check the platform and only enable these tests when `os.platform() !== 'win32'`?

Reply:
I see how you are doing win32 tests in the index.spec.ts now and will follow that